scripts/runtime-code-shasum.sh

```diff
@@ -17,6 +17,13 @@ fi
 export TEST_NODE_BLOCKTIME=1000
 export TEST_PROPOSALS_PARAMETERS_PATH="./tests/integration-tests/proposal-parameters.json"
 
+if [[ -z "$ALL_PROPOSALS_PARAMETERS_JSON" ]]; then
+  PROPOSALS_PARAMETERS_FILE=""
+else
+  PROPOSALS_PARAMETERS_FILE="./tmp.proposal_parameters.json"
```

Comment:
Saving the file as `tmp.proposal_parameters.json` on the target machine so that it doesn't interfere with git. Can be changed if required.

Reply:
Added a `runtime-inputs` dir as `gitignore` directory and the JSON file is stored here now.